### PR TITLE
Update Release Candidate Workflow

### PR DESCRIPTION
**Pull request by nelsitoPuglisi**
## Goal

Improve our release pipeline

## Testing

Not yet...

### CODE REVIEW

**Comment by nelsitoPuglisi** [comment]
* **#1004** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1004?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nelsitoPuglisi and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .github/workflows/build-rc-workflow.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ jobs:
 
       - name: Generate SDK RC - Publish and close Sonatype repository
         run: |
-          ./gradlew clean publishReleasePublicationToSonatype closeSonatypeStagingRepository -Dorg.gradle.parallel=false --stacktrace
+          ./gradlew clean check publishReleasePublicationToSonatype closeSonatypeStagingRepository -Dorg.gradle.parallel=false --stacktrace
```
Comment (nelsitoPuglisi) [inline .github/workflows/build-rc-workflow.yml:75]: Is the `check` required?

Comment (bidetofevil) [reply .github/workflows/build-rc-workflow.yml:75]: Only if we think the branch could contain bad code

**Review by bidetofevil — APPROVED** [review]
LGTM

**Pull request merged**